Sources/MessagingPush/PushHandling/iOSPushEventListener.swift

```diff
@@ -128,15 +128,6 @@ class iOSPushEventListener: NSObject, PushEventListener, UNUserNotificationCente
 
         guard !pushHistory.hasHandledPush(pushEvent: .didReceive, pushId: response.pushId, pushDeliveryDate: response.pushDeliveryDate) else {
             // push has already been handled. exit early
-
-            /*
```

Comment:
This comment got moved into the test class. I felt that it made more sense to have this comment explained in the test function that tests against this use case. So, if the test fails, you can read the comment that explains why it could be failing.